Makefile

```diff
@@ -24,6 +24,12 @@ VERIFY                      := true
 LEADER_ELECTION             := false
 IGNORE_OPERATION_ANNOTATION := true
 
+### GVisor version: https://github.com/google/gvisor/releases
+RUNSC_VERSION				 	:= 20200219.0
+
+### GVisor containerd shim version: https://github.com/google/gvisor-containerd-shim/releases
+CONTAINERD_RUNSC_SHIM_VERSION 	:= v0.0.4
```

Comment:
@ccwienk are these things to be expected in the component descriptor as well? These things are versions of binaries that will be copied to the shoot worker nodes if this extension is enabled.

Reply:
ping @ccwienk